qa/tasks/mgr/dashboard/test_rgw.py

```diff
@@ -202,6 +202,12 @@ def test_all(self):
             'tenant': JLeaf(str),
         }, allow_unknown=True))
 
+        # List all buckets names without stats.
+        data = self._get('/api/rgw/bucket?stats=false')
```

Comment:
```suggestion
        data = self._get('/api/rgw/bucket')
```

`stats` is False by default

Reply:
> `stats` is False by default

Yes, but this is already tested [above](https://github.com/ceph/ceph/pull/42865/files#diff-095a0da450fe86d821140fea33b38b4b7d92c7048549efc11346954c1b8f3144L186). I wrote this 1 more test just to check it with JS `false` value (as Python `False` is different from `'false'`) similar as tested with  `true`.